packages/configuration/src/index.ts

```diff
@@ -0,0 +1,36 @@
+import urlcat from 'urlcat'
+import { DEFAULT_HOST, DEFAULT_PREFIX } from './constants'
+
+class Configuration<T extends unknown> {
```

Comment:
Please make sure all tasks are scheduled in `packages/mask/background/tasks` and declare it in `packages/mask/src/extension/background-script/Jobs/index_hmr.ts`